tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -12766,9 +12766,6 @@ static StringRef ValidateAttributeStringArg(Sema &S, const AttributeList &A,
 
```

Comment:
Can we restore this entire function to the version in `main`? The `isCaseSensitive` flag was added for work graphs, so we should be able to just revert this function.

Reply:
I did exactly that in the first commit - it looks as if the subsequent merge reinstated the isCaseSensitive version without me noticing!
I will sort this on return from vacation...